ci-operator/config/stolostron/acmqe-autotest/stolostron-acmqe-autotest-main.yaml

```diff
@@ -0,0 +1,20 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+images:
+- dockerfile_path: ci/containerimages/fetch-managed-clusters/Dockerfile
+  to: fetch-managed-clusters
+promotion:
+  name: "2.7"
```

Comment:
@vboulos the acmqe-autotest is not versioning its branch like the others so we don't need to use 2.7 here, though I'm not sure of a suitable name to use. Do you have any suggestions?

Reply:
@ascerra yes no branch versioning for acmqe-autotest, let us use the main branch for this one

Reply:
This value doesn't represent the branch it represents the namespace name of the registry then the image name is the specific tag
`registry.ci.openshift.org/acm-qe/2.7:clc-ui-e2e`

Its kind of an odd syntax to me, most others do it this way I'm assuming because of these docs
https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/#publication-of-new-tags

We could put `main` in place of 2.7 then we will have
`registry.ci.openshift.org/acm-qe/main:fetch-managed-clusters`

I think the best solution would be to use `autotest` to be more descriptive about where this image is coming from
`registry.ci.openshift.org/acm-qe/autotest:fetch-managed-clusters`